standard_releases.md

```diff
@@ -1,59 +1,206 @@
+[github.com/openownership/data-standard](https://github.com/openownership/data-standard)
+as a naming convention for release branches, for example:
+
+* [github.com/openownership/data-standard/tree/v0-0-3](https://github.com/openownership/data-standard/tree/v0-0-3)
+* [github.com/openownership/data-standard/tree/v0-1](https://github.com/openownership/data-standard/tree/v0-1)
+
+These releases are published by enabling the release branch in ReadTheDocs, the
+"latest" tag in ReadTheDocs is manually changed to point at the latest version.
+
+**Note:** our release branch naming scheme is "vMAJOR-MINOR-PATCH"
+
+**Note:** we always include trailing zeroes in our version numbers. e.g. "v0-1-0" is
+preferred over "v0-1" because someone might interpret the latter as the latest
+version of v0.1.
+
+## Translations and releases
+
+Our standard assumption for translations is that they are generally
+**asynchronous with development**. That means that changes to the standard or
+documentation do not need to wait for translation before they can be released,
+they should be planned and scheduled as appropriate. For small changes, this
+most likely means batching and performing translations when a suitable number
+are required. For larger changes, particularly new releases, we're more likely
+to want to have all supported translations in place before releasing.
+
+## Release processes
+
+How exactly we perform a release depends on what is being released and which
+versions the changes affect. The sections below detail specific scenarios, but
+the following general questions always apply:
+
+1. Has the schema or any normative documentation changed?
+2. Does text need translating?
+3. Do the changes affect any existing, released versions?
+
+### A new release
+
+Preparing a new release is the most straightforward scenario, though not the
+least work! The general process is:
+
+* Decide on the set of changes that will be released. Some of these will have
+  likely already been merged in the course of normal development, but they may
+  need additional development, examples adding, etc.
+* Review, test and merge those changes into the `master` branch.
+* Depending on the other ongoing work, either call a "freeze" on the master
+  branch, whereafter merging changes that are not going to be part of the
+  release cannot be merged, or create a new release branch from master. A
+  release freeze is preferred, to reduce the complexity of incorporating any
+  release fixes back into master, but this can be weighed against the impediment
+  to other development on a case-by-case basis.
+* Begin translation of the new release (see [translations](/translations)) and
+  fixing of any snags or bugs. If `master` is frozen, these changes can be merged
+  into master, if using a release branch, merge them there for incorporation
+  into master after the release.
+* Once the release is finalised and everything is ready, release the new version
```

Comment:
* Once the release is finalised and everything is ready create a branch for the new version (eg, v0-2-0) from `master` 
* make the new version live on ReadTheDocs (I think we should some step-by-step instructions after this that are similar to the screenshotted step through on the translations page.